setup.py

```diff
@@ -1,8 +1,59 @@
-from setuptools import find_packages, setup
+import importlib
+import os
+import os.path as osp
+import subprocess
+
+from setuptools import Extension, find_packages, setup
+from setuptools.command.build_ext import build_ext
 
 __version__ = '0.0.0'
 URL = 'https://github.com/pyg-team/pyg-lib'
 
+
+class CMakeExtension(Extension):
+    def __init__(self, name, sourcedir=''):
+        Extension.__init__(self, name, sources=[])
+        self.sourcedir = os.path.abspath(sourcedir)
+
+
+class CMakeBuild(build_ext):
+    def get_ext_filename(self, ext_name):
+        # Remove Python ABI suffix:
+        ext_filename = super().get_ext_filename(ext_name)
+        ext_filename_parts = ext_filename.split('.')
+        ext_filename_parts = ext_filename_parts[:-2] + ext_filename_parts[-1:]
+        return '.'.join(ext_filename_parts)
+
+    def build_extension(self, ext):
+        import torch
+
+        extdir = os.path.abspath(osp.dirname(self.get_ext_fullpath(ext.name)))
+
+        if self.debug is None:
+            self.debug = bool(int(os.environ.get('DEBUG', 0)))
+
+        if not osp.exists(self.build_temp):
+            os.makedirs(self.build_temp)
+
+        cmake_args = [
```

Comment:
Do we need to disable `BUILD_TEST` for just `pip install`

Reply:
Yes, thank you :)